include/LDtkLoader/Entity.hpp

```diff
@@ -29,6 +29,9 @@ namespace ldtk {
         auto getGridPosition() const -> const IntPoint&;
         auto getColor() const -> const Color&;
         auto getPivot() const -> const FloatPoint&;
+        auto hasTile() const -> bool;
```

Comment:
New line before `hasSprite` method for clarity
```suggestion

        auto hasSprite() const -> bool;
```